.github/workflows/build.yml

```diff
@@ -185,9 +185,15 @@ jobs:
         uses: stCarolas/setup-maven@v4.5
         with:
           maven-version: ${{ env.MAVEN_VERSION }}
+      - name: Build & Test Java 8
+        if: ${{ matrix.java == 8}}
+        run: |
+          mvn -B "-Dbuild.repository.owner=${{ needs.setup.outputs.thisRepositoryOwner }}" "-Dbuild.repository.name=${{ needs.setup.outputs.thisRepositoryName }}" "-Dbuild.branch=${{ needs.setup.outputs.thisBranchName }}" "-Dbuild.number=${{ github.run_number }}" "-Dbuild.commit=${{ needs.setup.outputs.thisSha }}" "-DtrimStackTrace=false" -pl '!liquibase-dist,!liquibase-cdi-jakarta' clean test package surefire-report:report
       - name: Build & Test
```

Comment:
Can you change this step to be called something like "Build & Test all other Java versions"

Reply:
I'm calling it non-jdk-8 to be compliant with the pom profile